community/knative/README.md

```diff
@@ -38,7 +38,12 @@ knative
 - Knative requires a Kubernetes cluster v1.11 or newer with the
 [MutatingAdmissionWebhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller)
 enabled.
-- Istio - You should have Istio installed on your Kubernetes cluster. If you do not have it installed already you can do so by running the following commands:
+- Istio:
+  You may use an existing `ibm-istio` intance or install it through the catalog. To use `ibm-istio` with Knative you must apply the following command:
+```
```

Comment:
Can we say ICP catalog to be explicit.

Reply:
Trying to see if we can make it obvious that this is an additional step on top of installing ibm-istio and not the command to achieve that... Feel free to tweak..

Once you have installed ibm-istio, you need to add knative extras to make your instil installation work with Knative. Please use the below command to add the extras:

Reply:
We do not encourage using `ICP` externally but should always use `IBM Cloud Private` instead.